demo/test.html

```diff
@@ -137,6 +156,46 @@ <h3>Events</h3>
         });
       });
 
+      $("#trip-start-btn").click(function(e) {
+        e.preventDefault();
+        let externalId = $('#trip-id').val();
+        let mode = $('#trip-mode').val();
+        let destinationGeofenceTag = $('#trip-destination-tag').val();
+        let destinationGeofenceExternalId = $('#trip-destination-externalId').val();
+
+        const tripOptions = {
+          externalId,
+          mode,
+          destinationGeofenceTag,
+          destinationGeofenceExternalId,
+        }
+
+        Radar.startTrip(tripOptions, function(err, responseObj, response) {
```

Comment:
I know this is how nick designed it, but I'm not a fan of these global static functions that operate on an individual user level. something something concurrency

Reply:
it feels like `user.startTrip` may have been a more appropriate API

Reply:
anyways, nothing actionable here